playback/src/player.rs

```diff
@@ -1305,14 +1305,20 @@ impl PlayerInternal {
                                 // Engineering Society, 60, 399-408.
                                 if self.config.normalisation_method == NormalisationMethod::Dynamic
                                 {
-                                    // steps 1 + 2: half-wave rectification and conversion into dB
-                                    let abs_sample_db = ratio_to_db(sample.abs());
+                                    if !(self.normalisation_integrator > 0.0
+                                        || self.normalisation_peak > 0.0)
+                                    {
+                                        continue;
+                                    }
 
+                                    let ratio = sample.abs();
```

Comment:
To stay consistent with the rest of this method, I'd prefer `let sample_abs = sample.abs()` here.

Reply:
Sure whatever.

Reply:
You should stay consistent with your naming then also,lol. I called it `ratio` because the that's the name of the var in `ratio_to_db`.

Reply:
I certainly won't deny that. I learn as I go.